xml/System.Data.SqlTypes/SqlBoolean.xml

```diff
@@ -1529,7 +1529,7 @@
 ## Remarks  
  If either instance of `x` or `y` is null, the value of the <xref:System.Data.SqlTypes.SqlBoolean> will be null.  
   
- The equivalent method for this operator is <xref:System.Data.SqlTypes.SqlBoolean.Compare%28System.Data.SqlTypes.SqlBoolean%2CSystem.Data.SqlTypes.SqlBoolean%29?displayProperty=fullName>]]></format>
+ The equivalent method for this operator is <xref:System.Data.SqlTypes.SqlBoolean.op_Equality*></format>
```

Comment:
@GuardRex, definitely. The GreaterThan method is the preferred non-operator alternative to op_GreaterThan.